### PR TITLE
Add CuccoShackCuccoCount

### PR DESCRIPTION
**Pull request by garrettjoecox**
null

### Build Artifacts
  - [2ship-mac.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2236483057.zip)
  - [2ship-linux.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2236488093.zip)
  - [2ship-windows.zip](https://nightly.link/HarbourMasters/2ship2harkinian/actions/artifacts/2236499092.zip)

### CODE REVIEW

File: mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include <libultraship/bridge.h>
+#include "2s2h/GameInteractor/GameInteractor.h"
+
+extern "C" {
+#include "overlays/actors/ovl_En_Hs/z_en_hs.h"
+#include "overlays/actors/ovl_En_Nwc/z_en_nwc.h"
+
+void func_8095345C(EnHs* enHs, PlayState* play);
+void EnNwc_SpawnDust(EnNwc* enNwc, PlayState* play);
```
Comment (Archez) [inline mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp]: Do you need this dust func to be extern?

File: mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+            if (enHs->actionFunc != func_8095345C ||
+                CVarGetInteger("gEnhancements.Minigames.CuccoShackCuccoCount", 10) == 10) {
+                return;
+            }
+
+            // As soon as enough have grown up, set the count to the max
+            if (enHs->actor.home.rot.x >= (CVarGetInteger("gEnhancements.Minigames.CuccoShackCuccoCount", 10) * 2) &&
+                enHs->actor.home.rot.x != 20) {
+                enHs->actor.home.rot.x = 20;
```
Comment (Archez) [inline mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp:24]: lol at the game abusing rotation values.

**Review by Archez — APPROVED** [review]

File: mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp (hunk 1 above)
Comment (garrettjoecox) [reply mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp]: Oh nope this was from me playing with something

Comment (garrettjoecox) [reply mm/2s2h/Enhancements/Minigames/CuccoShackCuccoCount.cpp]:
```suggestion
```

**Review by balloondude2 — APPROVED** [review]
LGTM if a menu bar entry isn't needed.

**Pull request merged**